aws/account/terraform.tfvars.example

```diff
@@ -0,0 +1,11 @@
+prod_account_name="prod"
+prod_account_email="info+prod@cloudposse.co"
+
+staging_account_name="staging"
```

Comment:
I think we should maybe separate them into N files.

![image](https://user-images.githubusercontent.com/52489/41134750-e145e526-6a82-11e8-90c2-78dabf7ebeee.png)